### ISSUE TITLE
Java configuaration

### ISSUE THREAD

**Issue by ShubhamKalani** [mfornet/acmx#12]
how to configure it with java 13

**Comment by mfornet** [comment]
I was trying to do it for java, and current configuration doesn't allow to do it out of the box.
I see that for java we need an specific file name, and specific instructions to run the binary.

My plan to allow it will be through two new configurations:
- Custom source name (so you should type Main)
- Custom run line

So what I expect the configurations will look like to run a java program will be:
```
{
    "acmx.configuration.templatePath": "/path/to/java/template/Main.java",
    "acmx.configuration.extension": "java",
    "acmx.configuration.sourceName": "Main",
    "acmx.execution.compile": "javac -d $OUTPUT Main.java",
    "acmx.execution.run": "java -cp $OUTPUT Main",
}
```

**Comment by mfornet** [comment]
Closing in favor of #16

**Comment by mfornet** [comment]
@ShubhamKalani It was solved in #57 . Check [wiki](https://github.com/mfornet/acmx/wiki) to setup java.
